shell/src/main/java/alluxio/cli/fsadmin/report/MetricsCommand.java

```diff
@@ -112,19 +113,28 @@ public int run() throws IOException {
     return 0;
   }
 
+  /**
+   * Gets readable name from property name.
+   *
+   * @param key the property key to transfer
+   * @return a readable name
+   */
+  private String getReadableName(String key) {
```

Comment:
I would stick with using the exact metric name so that the metric has the same name everywhere. The current readability translation only partially works since it doesn't deal with camelCase or other abbreviations that might get used on the server side

Reply:
I think the current solution will work because we only use it to deal with a set of know property names. If some of the property names change, we will not print those properties in operation and rpc invocation sections and thus will not trigger getReadableName function.

Your suggestion also makes sense because the property names can just keep original and still readable. We actually do not need to transform them.

Thanks for your advice!